src/app/containers/sample.containers.js

```diff
@@ -0,0 +1,16 @@
+import { connect } from 'react-redux';
+import { sampleAction } from '../store/actions/sample.actions';
+// import your actual component here
+const SampleComponent = () => {};
```

Comment:
How about having an actual component here, this would inform folks on how to access the things they are passing into this component with `mapStateToProps` and `mapDispatchToProps`